modules/installation-azure-finalizing-encryption.adoc

```diff
@@ -68,7 +68,7 @@ The identity is in the format of `12345678-1234-1234-1234-1234567890`.
 [source,terminal]
 ----
 $ az role assignment create --assignee <cluster_service_principal_id> \// <1>
-     --role 'Contributor' \//
+     --role "<privileged_role>" \//
      --scope <disk_encryption_set_id> \// <2>
 ----
```

Comment:
We need to add a sequence number beside "<privileged_role>" , and write that can be provided here.

Reply:
In that case the live above this command also needs some modification 
```
Create a role assignment that grants the cluster service principal Contributor privileges to the disk encryption set by running the following command:
```

^^ Need to replace `Contributor` with appropriate word.

Reply:
Addressed